docs/configuration.md

```diff
@@ -474,6 +474,22 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.shuffle.service.enabled</code></td>
+  <td>false</td>
+  <td>
+    Enables the external shuffle service. This service preserves the shuffle files written by 
+    executors so the executors can be safely removed. This must be enabled if 
```

Comment:
we may want to add that the external shuffle service must be installed for this to work or point to docs describing that